docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -53,37 +40,27 @@ These instructions were tested with version 1.2.15 of the operator.
 
 :::
 
-Then, wait until the ArangoDB deployment is ready:
+### Optional: adjust configuration via a Helm Values File
```

Comment:
Can we add a link to the values.yaml readme?

Reply:
I link to the values.yaml readme 2 lines down.